App-Service-Environment-AzFirewall/azuredeploy.json

```diff
@@ -140,27 +143,28 @@
         },
         "aseSubnetServiceEndpoints": {
             "type": "array",
+            "defaultValue": [],
             "metadata": {
                 "description": "Service Endpoints enabled on the ASE subnet"
             }
         },
         "aseManagementIps": {
             "type": "array",
+            "defaultValue": [],
             "metadata": {
-                "description": "List of ASE management IPs"
+                "description": "List of ASE management IP addresses"
             }
         }
     },
     "variables": {
-        "vnetID": "[resourceId(resourceGroup().name, 'Microsoft.Network/virtualNetworks', parameters('vnetResourceName'))]",
+        "vnetID": "[resourceId('Microsoft.Network/virtualNetworks', parameters('vnetResourceName'))]",
```

Comment:
does the vnet have to be in the same resourceGroup?

Reply:
No - removed per your earlier comment.